core/search/fields.py

```diff
@@ -278,8 +294,20 @@ def get_condition(self, expression: Expression, remainder: List[str]) -> Any:
             )
 
         if isinstance(expression, Range):
-            if not (expression.include_low and expression.include_high):
+            if expression.high.value != "*" and not expression.include_high:
+                raise UnsupportedGrammarException("Exclusive range is not allowed for date-time field")
+            if expression.low.value != "*" and not expression.include_low:
                 raise UnsupportedGrammarException("Exclusive range is not allowed for date-time field")
```

Comment:
We still don't support include_high and include_low here: ranges are always inclusive. Should be left unchanged.